schema_salad/makedoc.py

```diff
@@ -227,7 +234,7 @@ def typefmt(self,
                 nbsp=False,             # type: bool
                 jsonldPredicate=None    # type: Optional[Dict[str, str]]
                 ):
-        # type: (...) -> Union[str, unicode]
+        # type: (...) -> Union[str, Text]
```

Comment:
@AleksandrSl Thanks for the clarifying this. :smile: 

I will make changes wherever required in PR https://github.com/common-workflow-language/schema_salad/pull/111